inventory/sample/group_vars/all/huaweicloud.yml

```diff
@@ -0,0 +1,17 @@
+## Values for the external Huawei Cloud Controller
+# external_huaweicloud_lbaas_subnet_id: "Neutron subnet ID to create LBaaS VIP"
+# external_huaweicloud_lbaas_network_id: "Neutron network ID to create LBaaS VIP"
+
+## Credentials to authenticate against Keystone API
+## All of them are required Per default these values will be
+## read from the environment.
+# external_huaweicloud_auth_url: "{{ lookup('env','OS_AUTH_URL')  }}"
```

Comment:
I see a lot of things related to OpenStack from a rapid search it seems to me that huawei cloud is somehow using OpenStack so are those really valid or is it just a copy paste somehow?

Reply:
It is really close to openstack since huaweicloud is based on openstack BUT it is not fully compatible. Especially the loadbalancers are not offering Octavia Support which is a MUST since k8s 1.26 with the openstack provider.  That's why huawei introduced it's own cloud provider as [kubernetes-sig](https://github.com/kubernetes-sigs/cloud-provider-huaweicloud) project.
But I get the point. Maybe we could use the existing OpenStack Provider and make the image field mutable so that one could use the huawei fork!?

Reply:
Thanks for the insight well I don't really mind just wanted to make sure it was intentional to leave the reference to Openstack :D